### DISCUSSION TITLE
Parquet export with encryption

### DISCUSSION THREAD

**Discussion by amoghnjoshi** [duckdb/duckdb#5058] [Q&A] [answered]
DuckDB's built-in parquet support is quite amazing. 
The parquet format supports encryption mentioned by https://github.com/apache/parquet-format/blob/master/Encryption.md. 
Is there any way (or future plan) to support import-export of the encrypted parquet files in DuckDB?

**Comment by hannes** [comment, marked as answer]
There are no plans at the moment

**Reply by amoghnjoshi** [reply]
ok, noted.
